cover.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+target_coverage=${1:-100}
+
+rm -f coverage.info
+# capture coverage info
+lcov --quiet --directory . --capture --output-file coverage.info
+# header files are not covered
+lcov --quiet --remove coverage.info "*/odpi/src/*.c" "*.h" --output-file coverage.info
+
+# Sample structure of lcov --summary coverage.info
+# Reading tracefile coverage.info
+# Summary coverage rate:
+#   lines......: 13.2% (142 of 1078 lines)
+#   functions..: 21.1% (12 of 57 functions)
+#   branches...: no data found
+#
+# grep to find the 'lines...' line with total % (e,g, 13.2 above)
```

Comment:
`TRAVIS` env variable will be set to true when running from travis-ci. This could be used to disable the part of the code not to be executed with travis.